TrafficCapture/captureOffloader/src/main/java/org/opensearch/migrations/trafficcapture/StreamChannelConnectionCaptureSerializer.java

```diff
@@ -112,7 +112,8 @@ private void writeObservationTag(int fieldNumber) throws IOException {
                 getWireTypeForFieldIndex(TrafficObservation.getDescriptor(), fieldNumber));
     }
 
-    private void beginWritingObservationToCurrentStream(Instant timestamp, int captureTag, int captureClosureSize) throws IOException {
+    private void beginSubstreamObservationToCurrentStream(Instant timestamp, int captureTag, int captureClosureSize) throws IOException {
```

Comment:
Isn't substream and observation synonmous, this throws me off a bit. Maybe `writeSubstreamBeginningToCurrentStream` and `writeSubstreamBeginningWithTagToCurrentStream` for later one? Even then the ToCurrentStream could possibly just be implied since everything in this file is working off of the current stream